gtda/homology/simplicial.py

```diff
@@ -721,3 +722,272 @@ def plot(Xt, sample=0, homology_dimensions=None):
+        :math:`p` equals `coeff`.
+
+    max_edge_weight : float, optional, default: ``numpy.inf``
+        Maximum edge weight to be considered in the filtration. All edge
+        weights greater than that value will be considered as
+        infinitely-valued, i.e., absent from the filtration and topological
+        features at scales larger than this value will not be detected.
+
+    infinity_values : float or None, default : ``None``
+        Which death value to assign to features which are still alive at
+        filtration value `max_edge_weight`. ``None`` means that this
+        death value is declared to be equal to `max_edge_weight`.
+
+    max_entries : int, optional, default: ``-1``
+        Number controlling the degree of precision in the matrix
+        reductions performed by the the backend. Corresponds to the parameter
+        ``approximation`` in :func:`pyflagser.flagser`. Increase for higher
+        precision, decrease for faster computation. A good value is often
+        ``100000`` in hard problems.  A negative value computes highest
+        possible precision.
+
+    n_jobs : int or None, optional, default: ``None``
+        The number of jobs to use for the computation. ``None`` means 1 unless
+        in a :obj:`joblib.parallel_backend` context. ``-1`` means using all
+        processors.
+
+    Attributes
+    ----------
+    infinity_values_ : float
+        Effective death value to assign to features which are still alive at
+        filtration value `max_edge_weight`.
+
+    See also
+    --------
+    VietorisRipsPersistence, SparseRipsPersistence, EuclideanCechPersistence, \
+    ConsistentRescaling, ConsecutiveRescaling
+
+    Notes
+    -----
+    The `pyflagser <https://github.com/giotto-ai/pyflagser>`_ Python package
+    is used for binding `Flagser <https://github.com/luetge/flagser>`_, a C++
+    backend for computing the (persistent) homology of (filtered) directed
+    flag complexes.
+
+    For more details, please refer to the `flagser documentation \
+    <https://github.com/luetge/flagser/blob/master/docs/\
+    documentation_flagser.pdf>`_.
+
+    Persistence diagrams produced by this class must be interpreted with
+    care due to the presence of padding triples which carry no information.
+    See :meth:`transform` for additional information.
+
+
+    References
+    ----------
+    [1] D. Luetgehetmann, D. Govc, J. P. Smith, and R. Levi, "Computing \
+        persistent homology of directed flag complexes", Algorithms,
+        13(1), 2020.
+
+    """
+
+    _hyperparameters = {
+        'homology_dimensions': {
+            'type': (list, tuple), 'of': {
+                'type': int, 'in': Interval(0, np.inf, closed='left')}},
+        'directed': {'type': bool},
+        'coeff': {'type': int, 'in': Interval(2, np.inf, closed='left')},
+        'max_edge_weight': {'type': Real},
+        'infinity_values': {'type': (Real, type(None))},
+        'max_entries': {'type': int}
```

Comment:
Add list of algorithms? (See comment above.)

Reply:
The validaty of the filtration algorithm is already checked by `pyflagser`.

Reply:
Thanks!